src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.cs

```diff
@@ -7,7 +7,8 @@
 
 namespace System.Numerics
 {
-    /// <summary>A structure encapsulating a 3x2 matrix.</summary>
+    /// <summary>Represents a 3x2 matrix.</summary>
+    /// <remarks>[!INCLUDE[vectors-are-rows-paragraph](~/includes/system-numerics-vectors-are-rows.md)]</remarks>
```

Comment:
@gewarren that's a good point. In the first PR, I was asked to remove the CDATA since it was not providing any value in triple slash comments, was using a lot of unnecessary space and polluting the code visually.

So I have a question for you to help address this: When we send intellisense xmls to the Docs build system, is it smart enough to automatically wrap non-empty remarks (including remarks that say `To be added.`) with these tags?:

 `<format type="text/markdown><![CDATA[ ... ]]></format>` 

If not, then we have two options:

A) Convert markdown links like these to valid xml. Do you know what's the non-markdown syntax for a link to an md file? And do you know if the Docs build system will be able to read that correctly and render the md document properly?
B) Talk to the Docs dev team to request converting any remarks we send to them to markdown (if they don't do it yet).

Thoughts?

Reply:
Another thing to consider is that the xmls that Roslyn will produce after the source of truth are comments in source code, we will be shipping that as part of the intellisense xml files right? How will this look when a customer hovers over an API on VS and VS takes the info for that API from comments with these includes or `[CDATA]`?

Reply:
AFAIK, remarks do not show up in VS intellisense in any way.

Reply:
> Do you know what's the non-markdown syntax for a link to an md file?

I don't know. And the thing is, you don't just want a link, you actually want to import the contents of the linked file.

> When we send intellisense xmls to the Docs build system, is it smart enough to automatically wrap non-empty remarks?

No, it does not. See for example [here](https://github.com/dotnet/dotnet-api-docs/blob/733b2cecdd45ab3ed5efee2f001d9b07730f06e1/xml/Microsoft.Extensions.Diagnostics.HealthChecks/HealthCheckPublisherOptions.xml#L117).

Reply:
@gewarren to address this problem, we can keep the CDATA section for remarks that require to show an include.

Reply:
That's what the machinelearning repo does for its linked code snippets. See https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs#L30.